lua/usip_player.lua

```diff
@@ -133,6 +133,58 @@ tParserCommandUsip:flag('--no_reset'
 ):description('Skip the last reset after booting an USIP. Without the reset, verifying the content is also disabled.'
 ):target('fDisableReset'):default(false)
 
+
+-- NXTFLASHER-603
+-- NXTFLASHER-550
+
+-- Add the "disable_security" command and all its options.
+strDisableSecurityHelp = [[
+    Disable security settings at COM and APP SIPs.
+
+    Following parameter will be set:
+    COM SIP:
+    - Security Access Level (SAL): OFF
+    - Secure Boot Mode (SBM) :     OFF
+    - SIP will be copied
+    - SIP will be visible
+    - (ENABLE_MI_UART_IN_SECURE :  OFF)  reserved for netX 90 rev2
+
+    APP SIP:
+    - Security Access Level (SAL): OFF
+    - Secure Boot Mode (SBM) :     OFF
+    - SIP will be copied
+    - SIP will be visible
+    - (ASIG_SIGNED_BINDING :       OFF) reserved for netX 90 rev2
+
+]]
+
+local tParserCommandDisableSecurity = tParser:command('ds disable_security', strDisableSecurityHelp):target('fCommandDisableSecurity')
+tParserCommandDisableSecurity:option(
+    '-V --verbose'
+):description(
+    string.format(
+        'Set the verbosity level to LEVEL. Possible values for LEVEL are %s.', table.concat(atLogLevels, ', ')
+    )
+):argname('<LEVEL>'):default('debug'):target('strLogLevel')
+tParserCommandDisableSecurity:option('-p --plugin_name'):description("plugin name"):target('strPluginName')
+tParserCommandDisableSecurity:option('-t'):description("plugin type"):target("strPluginType")
+tParserCommandDisableSecurity:flag('--no_verify_USIP_sig'):description(
+    "Do not verify the signature of the usip images against a netX; if the signature does not match, cancel the process!"
+):target('fVerifyUsipSigDisable')
+tParserCommandDisableSecurity:flag('--no_verify_SIP_content'):description(
+    "Do not verify the content of an usip image against a netX SIP content after writing the usip."
+):target('fVerifySipContentDisabled')
+tParserCommandDisableSecurity:option('--bootswitch'):description(strBootswitchHelp):target('strBootswitchParams')
+-- todo add more help here
+tParserCommandDisableSecurity:option('--sec'):description("Path to signed helper image directory"):target('strSecureOption'):default(tFlasher.DEFAULT_HBOOT_OPTION)
+tParserCommandDisableSecurity:option('--sec_phase2 --sec_p2'):description(strHelpSecP2):target('strSecureOptionPhaseTwo'):default(tFlasher.DEFAULT_HBOOT_OPTION)
```

Comment:
Why do we need --sec_phase2?

Reply:
Since we only disable the secure boot, NO NEED to support a different helper file set with a different key or unsigned. A correct signed helper file will be accepted anyway, because the keys are not chenged with these USIP files.